modules/conversion-center/module.php

```diff
@@ -80,6 +81,9 @@ public function __construct() {
 			$documents_manager->register_document_type( self::CONTACT_PAGE_DOCUMENT_TYPE, Contact_Buttons::get_class_full_name() );
 		} );
 
+		add_action( 'wp_ajax_elementor_send_clicks', [ $this, 'handle_click_tracking' ] );
+		add_action( 'wp_ajax_nopriv_elementor_send_clicks', [ $this, 'handle_click_tracking' ] );
```

Comment:
Where is send_clicks being called from in the JS?

Reply:
@louiswol94 this is from pro but it will be also from core javascript

Reply:
@nicoladj77 Will you add it in a future PR or what is the plan?

Reply:
@louiswol94 I have already a PR ready it was just to keep PRs small 😄